gdx/src/com/badlogic/gdx/graphics/glutils/ImmediateModeRenderer.java

```diff
@@ -18,8 +18,9 @@
 
 import com.badlogic.gdx.graphics.Color;
 import com.badlogic.gdx.math.Matrix4;
+import com.badlogic.gdx.utils.Disposable;
 
-public interface ImmediateModeRenderer {
+public interface ImmediateModeRenderer extends Disposable {

```

Comment:
Unlike Screen, this is probably OK. I doubt much existing code implements this interface.

Reply:
Even it was, programmer (That I use) should able to fix if even a case.